### PR TITLE
Add Czech localization

### PR DESCRIPTION
**Pull request by ppetr**
One item is only guessed (marked with a comment), my device doesn't report it.

I also couldn't find any language files to extract in the latest firmware update 2.88.2-9488.

### CODE REVIEW

File: luxwslang/czech.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	NavSwitchOffs:   "Odepnutí",
+
+	NavOpHours:      "Provozní hodiny",
+	HoursImpulsesRe: regexp.MustCompile(`^Počet startů\s`),
+
+	NavSystemStatus:       "Status zařízení",
+	StatusType:            "Typ TČ",
+	StatusSoftwareVersion: "Softwarová verze",
+	StatusOperationMode:   "Provozní stav",
+	StatusPowerOutput:     "Výkon", // TODO: Not verified.
```
Comment (hansmi) [inline luxwslang/czech.go:30]: The string appears to be correct. I have an older version of the software archived and used [wp2reg-language-extractor](https://github.com/hansmi/wp2reg-language-extractor/) to get the English, German and Czech strings:

```
928,actual capacity,Leistung Ist,Výkon
```

Do you want to remove the TODO before merging?

**Review by hansmi — COMMENTED** [review]
Thank you a lot for your contribution. Just one comment.

**Comment by hansmi** [comment]
> I also couldn't find any language files to extract in the latest firmware update 2.88.2-9488.

Unfortunately they started embedding the translations in the main binary in newer versions of the software. It's still possible to extract them, just not as easily as from the previously separate files.

File: luxwslang/czech.go (hunk 1 above)
Comment (ppetr) [reply luxwslang/czech.go]: Thank you for verifying this, removed.

Comment (hansmi) [reply luxwslang/czech.go]: Thank you. I squashed the commits to have a clean history (there were some `HG:` artifacts too) and merged everything.

**Pull request merged**